src/homePage/homePage.component.tsx

```diff
@@ -0,0 +1,326 @@
+                  >
+                    {'Explore'}
+                  </Button>
+                </Box>
+              </Box>
+            </Paper>
+          </Grid>
+          <Grid item sm={12} md={4}>
+            <Paper sx={paperStyles} elevation={1}>
+              <Box sx={paperContentStyles}>
+                <PaperHeading variant="h4">{'Manufacturers'}</PaperHeading>
+                <PaperDescription variant="body1">
+                  {'View the manufacturers of parts used by EPAC.'}
+                </PaperDescription>
+                <Box marginTop="auto">
+                  <Button
+                    color="primary"
+                    variant="contained"
+                    component={Link}
+                    to={'/manufacturer'}
+                    data-testid="manufacturer-button"
+                  >
+                    {'Explore'}
+                  </Button>
+                </Box>
+              </Box>
+            </Paper>
+          </Grid>
+          <Grid item sm={12} md={4}>
+            <Paper
+              sx={{ ...paperStyles, backgroundColor: '#003088' }}
+              elevation={1}
+            >
+              <div
+                style={{
+                  backgroundImage: `url(${props.greenSwirl2Image})`,
+                  backgroundRepeat: 'no-repeat',
+                  backgroundPosition: 'top right',
+                  backgroundSize: 'auto 100%',
+                  height: '100%',
+                }}
+              >
+                <Box sx={paperContentStyles}>
+                  <BluePaperHeading variant="h4">{'EPAC'}</BluePaperHeading>
+                  <BluePaperDescription color="#ffffff" variant="body1">
+                    {
+                      'World-leading centre for scientific and industrial research.'
+                    }
+                  </BluePaperDescription>
+                  <Box marginTop="auto">
+                    <LightBlueButton
+                      color="primary"
+                      variant="contained"
+                      href={
+                        'https://www.clf.stfc.ac.uk/Pages/EPAC-Applications.aspx'
+                      }
+                      data-testid="facility-button"
+                    >
+                      {'Read more'}
+                    </LightBlueButton>
+                  </Box>
+                </Box>
+              </div>
+            </Paper>
+          </Grid>
+        </Grid>
+      </Box>
+    </div>
+  );
+};
+
+export const HomePage = React.memo((): React.ReactElement => {
+  const HomePage = BaseHomePage;
+  return (
+    <HomePage
+      logo={DGLogo}
+      backgroundImage={BackgroundImage}
+      greenSwirl1Image={GreenSwirl1Image}
+      greenSwirl2Image={GreenSwirl2Image}
+      decal1Image={Decal1Image}
+      decal2Image={Decal2Image}
+      decal2DarkImage={Decal2DarkImage}
+      decal2DarkHCImage={Decal2DarkHCImage}
+      facilityImage={FacilityImage}
```

Comment:
I tried testing whether we needed `pluginHost` here like in https://github.com/ral-facilities/datagateway/blob/develop/packages/datagateway-dataview/src/page/translatedHomePage.component.tsx and originally it seemed we didn't when I used the production docker compose file, but when I tested it locally in development it seems we actually do as replacing one of the images and running it in SciGateway gives
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/41ff85a7-6e39-4c14-8ce2-d9e9c9e06e8e), but adding `"http://localhost:5001" +` to the image works. So I think we are going to need the same here afterall.

Also the button colours are wrong in dark mode in this image due to https://github.com/ral-facilities/inventory-management-system/blob/a15cde2807168b692917ac8628e6f08dd2c45419/src/imsThemeProvider.component.tsx#L29-L37. @joshuadkitenge Why was this done?

Reply:
Spoke to Joshua about the button colours, its like this as it was changed for OperationsGateway - I will make a note on #152 to address it later as I think it will require a SciGateway change.